Minor typo fixes

abhinavsp0730 opened the pull request.

abhinavsp0730 commented:
Hey, 
@wolffg, @MarkDaoust and @yashk2810 kindly review it.

abhinavsp0730 commented:
Hey, @shivaylamba I've done some changes to resolve the conflicts kindly review it now.

shivaylamba commented:
> Hey, @shivaylamba I've done some changes to resolve the conflicts kindly review it now.
@abhinavsp0730 There are still merge conflicts.

abhinavsp0730 commented:
Hey, @shivaylamba I reverted my previous commit and just corrected some words. 
And not able to find what is causing merge conflicts. @shivaylamba would you please help me 
figuring out what is causing merge conflicts, so that I'll resolve it.

MarkDaoust commented:
@abhinavsp0730  Can you fix the merge conflicts?

abhinavsp0730 commented:
Closing this PR because it has already fixed.
